assets/test/hooks/session/cursor_history.test.js

```diff
@@ -0,0 +1,99 @@
+import CursorHistory from "../../../js/hooks/session/cursor_history";
+
+test("goBack returns when there's at least one cell", () => {
+  const cursorHistory = new CursorHistory();
+  const expectedEntry = { cellId: "123", line: "1", column: "1" };
+
+  expect(cursorHistory.canGoBack()).toBe(false);
+
+  cursorHistory.entries = [expectedEntry];
+  cursorHistory.index = 0;
+
+  expect(cursorHistory.index).toBe(0);
```

Comment:
I think both entries and index is internal state, so we shouldn't write/read it (similarly to how we wouldn't write/read GenServer state). Instead, we want to use the public APIs to setup the state.

So here specifically we can just push instead. Same in the tests below :)